go.mod

```diff
@@ -0,0 +1,3 @@
+module k8s.io/kubeadm
+
+go 1.12
```

Comment:
Do we want this toplevel `go.mod`?

Reply:
we should't need it, as long as the sub-folders have a go.mod.

Reply:
We don't care about this file, but it is generated by go every time you do an operation involving go modules in kinder or in the operator folder

Reply:
> We don't care about this file, but it is generated by go every time you do an operation involving go modules in kinder or in the operator folder

that's not entirely true.
if you run `cd kinder && ./hack/verify-deps.sh` the file will not be generated.

`/` itself is not a module and we should avoid treating it as such.

Reply:
what operations are causing the file generation?

Reply:
I find it added every time a new dependency is added